Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM haxe:4.1-alpine
-
-# RUN apk update && apk upgrade && \
-#     apk add --no-cache bash git openssh && \
-#     apk add wget git
+FROM haxe:4.2-alpine
 
 COPY . /opt/test-runner
 WORKDIR /opt/test-runner
 RUN yes | haxelib install all
 RUN haxe build.hxml
-
-ENTRYPOINT ["/opt/test-runner/bin/run.sh"]
\ No newline at end of file
+RUN apk add --no-cache bash
```

Comment:
Was there a reason you moved this below the RUN instructions above it? I think it might make sense to have this above those lines as the `apk add` command can then be easily cached by Docker.